gnomad/utils/annotations.py

```diff
@@ -1519,3 +1529,308 @@ def _create_group_dicts(
 
     # Returns the constructed dictionary.
     return final_freq_dict
+
+
+def gks_compute_seqloc_digest(vrs_variant: dict) -> dict:
+    """
+    Compute and set the digest-based id for the sequence location.
+
+    Take a dict of a VRS variant that has a sequence location that does not yet
+    have the digest-based id computed. Computes it and assigns it to .location._id.
+
+    :param vrs_variant: VRS variant dict
```

Comment:
could provide more detail here on how the dictionary is expected to be structured

Reply:
added based on https://github.com/theferrit32/gnomad_methods/blob/3523684d048d4a64fd089f48e8232f3129d73651/gnomad/utils/annotations.py#L1364; let me know how this looks and if the types I added are right!